binding/jms/src/test/java/io/tracee/binding/jms/TraceeMessageListenerTest.java

```diff
@@ -29,7 +28,8 @@
 	private final TraceeMessageListener unit = new TraceeMessageListener(backend);
 	private final InvocationContext invocationContext = mock(InvocationContext.class);
 	private final Message message = mock(Message.class);
-	private final Map<String,String> encodedContext = new HashMap<String, String>();
+	private final Map<String, String> encodedContext = new HashMap<String, String>();
+	private MdbLike mdbLike = spy(new MdbLike());
```

Comment:
Nice way to double-check the usage of the listener. Too bad for Mockito not able to find the default constructor when switching to  `spy(MdbLike.class)` :-)
:+1: